README.md

```diff
@@ -1,13 +1,55 @@
 # Texas Tribune Files App
 
-This app powers the frontend of the Texas Tribune's CMS file management system. It provides a  flexible [Web Component](https://developer.mozilla.org/en-US/docs/Web/Web_Components) based API for creating a file browser within a web browser.
+This app powers the frontend of the Texas Tribune's CMS file management system. It provides a flexible [Web Component](https://developer.mozilla.org/en-US/docs/Web/Web_Components) based API for creating a file browser within a web browser.
 
 [Documentation](https://texastribune.github.io/files)
 
-## Development
+## Usage
 
-Run `npm install` to install dependencies.
+To create a simple in-memory file browser, use the following custom elements on your webpage:
 
-This repository uses `tsc` for code compilation. Run `npm run build` to build `src` into `lib`.
+```
+<script src="https://texastribune.github.io/files/dist/bundle.js"></script>
+
+<file-browser select-multiple>
+  <memory-directory name="memory"></memory-directory>
+  <file-browser-context-menu></file-browser-context-menu>
+</file-browser>
+```
+
+To create an S3 bucket file browser, use the following custom elements on your webpage:
+
+```
+<script src="https://texastribune.github.io/files/dist/bundle.js"></script>
+
+<file-browser select-multiple>
+    <s3-directory
+      url="http://static.texastribune.org/"
+      prefix="graphics/"
+      max-keys="100"
+    ></s3-directory>
+  <file-browser-context-menu></file-browser-context-menu>
+</file-browser>
+```
```

Comment:
maybe for this one, we could generalize the domain and prefix a little? I'm just thinking that since these won't work on any ol' webpage, something like yourS3bucket.com is more accurate and as a bonus less obvious where we store the goods with this being a public repo @SimmonsRitchie